use/wordcount-example.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: default
+title: "Beam WordCount Example"
+permalink: /use/wordcount-example/
+redirect_from: /docs/use/wordcount-example/
+---
+
+# Apache Beam WordCount Example
+
+> **Note:** This walkthrough is still in progress. There is an open issue to finish the walkthrough ([BEAM-664](https://issues.apache.org/jira/browse/BEAM-664)).
+
+The WordCount examples demonstrate how to set up a processing pipeline that can read text, tokenize the text lines into individual words, and perform a frequency count on each of those words. The Beam SDKs contain a series of these four successively more detailed WordCount examples that build on each other. The input text for all the examples is a set of Shakespeare's texts.
+
+Each WordCount example introduces different concepts in the Dataflow SDKs. Begin by understanding Minimal WordCount, the simplest of the examples. Once you feel comfortable with the basic principles in building a pipeline, continue on to learn more concepts in the other examples.
+
+* **Minimal WordCount** demonstrates the basic principles involved in building a Dataflow pipeline.
+* **WordCount** introduces some of the more common best practices in creating re-usable and maintainable pipelines.
+* **Debugging WordCount** introduces logging and debugging practices.
+* **Windowed WordCount** demonstrates how you can use Dataflow's programming model to handle both bounded and unbounded datasets. Note: Windowed WordCount is not yet available in the Python SDK.
+
+## MinimalWordCount
+
+Minimal WordCount demonstrates a simple pipeline that can read a block of text from Google Cloud Storage, apply transforms to tokenize and count the words, and write the data to an output text file. This example hard-codes the locations for its input and output files and doesn't perform any error checking; it is intended to only show you the "bare bones" of creating a Beam pipeline. In later examples, we will parameterize the pipeline's input and output sources and show other best practices.
```

Comment:
In the Beam version, GCS is an implementation detail, not a concept we want to teach. So read from a text file... blah blah blah. The default input is stored in a world-readable bucket in...